native/src/components/PoiDetails.tsx

```diff
@@ -55,7 +55,7 @@ const PoiDetails = ({ poi, language, distance }: PoiDetailsProps): ReactElement
     appointmentUrl ?? poi.contacts.find(contact => contact.website !== null)?.website ?? null
 
   return (
-    <PoiDetailsContainer>
+    <PoiDetailsContainer accessibilityLabel={`${title} - ${category.name}`}>
```

Comment:
:thinking: Could you explain why you decided to add this as a a11y label instead of an empty string?
Isn't this just a duplicate of the two strings set below which are read in any case?

Reply:
what I found is it will skip the headers that in the Poi Details.. did you give it a test? not sure how it will work for iOS though...

Reply:
I sadly can't test this atm as my phone is google free and my emulators don't run atm :exploding_head: Sorry :/
So just to confirm: What exactly is skipped? Only the title? Or also the category? Is the rest of the content read correctly? And do you have any clue why that happens?

Reply:
Here's an example of with this line and without it:
https://github.com/user-attachments/assets/27a1cf54-e739-43a6-a537-3d401e88ca9b

Reply:
Thanks, so the change is basically to automatically read this once a poi is selected? Sounds good to me.

Reply:
@LeandraH Can you test how it behaves on iOS please?

Reply:
Sorry, it's too annoying to switch branches from the update branch, I'm not doing any code reviews at the moment. But I think @f1sh1918 also has an iPhone?